src/biogeochem/CropType.F90

```diff
@@ -972,4 +968,31 @@ subroutine checkDates( )
 
   end subroutine checkDates
 
+  real(r8) function latbaset(baset, latdeg, baset_latvary_intercept, baset_latvary_slope)
+    ! !ARGUMENTS:
+    real(r8), intent(in) :: baset
+    real(r8), intent(in) :: latdeg
+    real(r8), intent(in) :: baset_latvary_intercept
+    real(r8), intent(in) :: baset_latvary_slope
+
+    ! Was originally
+    !     maxlat = latbaset_max_lat(baset_latvary_intercept, baset_latvary_slope)
+    !     if (abs(latdeg) > maxlat) then
+    !         latbaset = baset
+    !     else
+    !         latbaset = baset + baset_latvary_intercept - baset_latvary_slope*abs(latdeg)
+    !     end if
+    ! But the one-liner below should improve efficiency, at least marginally.
+
+    latbaset = baset + baset_latvary_intercept - min(baset_latvary_intercept, baset_latvary_slope * abs(latdeg))
+
+  end function latbaset
+
+  real(r8) function latbaset_max_lat(intercept, slope)
```

Comment:
Is this function only used by the unit tests? If so, I recommend adding a comment about that, so nobody deletes this function thinking that it serves no purpose.

Reply:
Good point! In 239aa87, I moved it to the test module that uses it.